Voice Calculator/voiceCalc.py

```diff
@@ -0,0 +1,103 @@
+import os
+import pyttsx3
+import speech_recognition as sr
+import tkinter.messagebox as tmessage
+import wolframalpha
+
+from os.path import exists
+
+listener = sr.Recognizer()
+engine = pyttsx3.init()
+voices = engine.getProperty('voices')
+engine.setProperty('voice', voices[0].id)
+wolfprimeaplahe_app = input('Enter the API Token')
+
+
+def audio(audio):
+    engine.say(audio)
+    engine.runAndWait()
+
+
+def welcomeInst():
+    print('Welcome to Calculator :)')
+    audio('Welcome to Calculator :)')
+    print('If you want calculate something please tell calcualte and then your expression')
+    audio('If you want calculate something please tell calcualte and then your expression')
+    print('For example CALCULATE 7 PLUS 8 or CALCULATE sin30 plus cot20')
+    audio('For example CALCULATE 7 PLUS 8 or CALCULATE sin30 plus cot20')
+
+
+def _takeCommand():
+    r = sr.Recognizer()
+    with sr.Microphone() as source:
+        print("Listning....")
```

Comment:
```suggestion
        print("Listening....")
```

Also, use audio function to speak out this "Listening"